spec/models/taxon_spec.rb

```diff
@@ -46,6 +46,38 @@
     end
   end
 
+  context "without url_override set" do
+    it "returns an empty string to comply with the schema definition" do
+      taxon = described_class.new
+
+      expect(taxon.url_override).to eq("")
```

Comment:
If we had this as `nil` would we be able to not set it at all in the content item? Ah, but it's being saved and retrieved from publishing-api, so that's not possible in this app, right?

I note that in https://github.com/alphagov/govuk_publishing_components/pull/2093, it would work the same whether the field is nil or an empty string because we're using `url_override.present?`, so it doesn't particularly matter on the frontend